privacyidea/lib/smtpserver.py

```diff
@@ -103,9 +103,15 @@ def test_email(config, recipient, subject, body, sender=None,
         msg['To'] = ",".join(recipient)
         msg['Date'] = strftime("%a, %d %b %Y %H:%M:%S +0000", gmtime())
         msg['Reply-To'] = reply_to
-
-        mail = smtplib.SMTP(config['server'], port=int(config['port']),
-                            timeout=config.get('timeout', TIMEOUT))
+        if config['server'].startswith('smtps://'):
+            mail = smtplib.SMTP_SSL(config['server'][8:], port=int(config['port']),
```

Comment:
Please add a test case (mock) for this!
Is it necessary to add some docs?

Have you checked if `SMTP_SSL` exists in Python 2.7.5?

Reply:
Yes it is also in 2.7.